src/components/common/container.module.scss

```diff
@@ -17,5 +17,8 @@
   }
   &.container-narrow {
     max-width: 900px;
+    @media (min-width: $viewport-lg) {
```

Comment:
I think we should leave the margin where it is.

Reply:
I updated to remove this.

Reply:
Thanks!